docs/platform_ccl.rst

```diff
@@ -0,0 +1,458 @@
+.. _cclplatform:
+
+CC-Light Platform Configuration File
+------------------------------------
+
+:Note: The platform comfiguration file for its structure is platform independent. So, either the text below is split into a platform independent structure definition (and put one level higher) and a platform dependent content, or the whole thing is copied for other platforms and then adapted (which is not wise considering updates).

```

Comment:
👍

Reply:
I think it is a good idea to describe the structure of these files in the generic platform docs before going into the specifics of each backend in the specific sections. Let's mark this as a todo and not make it block this PR.